stripe/lib.w

```diff
@@ -0,0 +1,126 @@
+bring util;
+bring cloud;
+bring "./types.w" as types;
+
+/// Properties for Stripe webhook
+pub struct Props {
+  /// The secret key for Stripe as a string
+  secretKey: str?;
+  /// Secret key for Stripe as a Secret resource (recommended)
+  secretKeySecret: cloud.Secret?;
```

Comment:
Why do we need both options?

Reply:
Looked at the OpenAPI example, I did mainly for testing to be honest. Being able to create a Stripe client without secrets defined just passing a string in, seems odd though.

I need to explore how tests mock the Secret resource....and understand this more.